src/components/DiscussionForm.jsx

```diff
@@ -142,6 +142,8 @@ class DiscussionForm extends Component {
 
     return (
       <article className="discussion-form-section">
+        <h3 className="form-directions">Start a discussion.</h3>
+        <p className="form-directions">Be clear and specific so others can be as helpful as possible! After writing your question, you will be prompted to select a CCSS that aligns to your question.</p>
```

Comment:
These clear instructions will be helpful to get what we want in the discussions from the user.